lib/src/obsoleted/reporters/html/html_reporter.dart

```diff
@@ -52,7 +52,7 @@ class ReportTableRecord {
   final String title;
   final String link;
 
-  final FileReport report;
+  final metrics.FileReport report;
```

Comment:
Will this one be removed/renamed later (having two classes with the same name looks confusing)?

Reply:
sure